### ISSUE TITLE
Problem with Layout Colour black

### ISSUE THREAD

**Issue by huwred** [UmTemplates/UmBootstrap#231]
if you select --bs-black-rgb as a background colour it still does not change the background to black.

This is because it appends the shade value i.e. -bs-black-rgb-500 but there are no styles defined for these black shades

**Comment by deanleigh** [comment]
That is correct - Currently Bootstrap do not provide shades for Black and White as they already have some greys instead.
Of course if -500 is black it can't get any darker and if -500 is white it can't get any lighter

They are also in the process of changing quite a lot around the colour system.
However, I could:

- Make black and white -5oo
- Make black and white remain as such across all shades
- make a grey -500 that goes to black and white at the extremes, but I think this may be confusing with the greys

Of course the Black for the Area Colour does work as they are special colours that also have a text colour of the correct contrast in the sass

![image](https://github.com/UmTemplates/UmBootstrap/assets/6745929/e960cacc-990a-457d-875e-f7506869dfc9)

 I will take a look at one of the options I mentioned.

**Comment by huwred** [comment]
I just created myself some black-rgb-5oo to work around it for now :)
